src/libs/utils/arrangeByLanguage.ts

```diff
@@ -0,0 +1,13 @@
+import * as R from 'ramda';
+import { SupportedLanguage } from '../i18n';
+import { Contributor } from '../domain/contributor';
```

Comment:
since you're importing a lot of domain related code... mb your code is also part of the domain :thinking:

Reply:
hum, it's especially useful but there is no domain logical, no ?